gluten-data/src/main/scala/org/apache/spark/shuffle/ColumnarShuffleWriter.scala

```diff
@@ -84,7 +84,7 @@ class ColumnarShuffleWriter[K, V](
 
   private var nativeShuffleWriter: Long = -1L
 
-  private var splitResult: SplitResult = _
+  private var splitResult: GlutenSplitResult = _
```

Comment:
Why rename `SplitResult` to `GlutenSplitResult`? @zhztheplayer has removed the prefix `Gluten` from other class

Reply:
It's a new subclass of SplitResult. Because CH backend now has `CHSplitResult` that inherits `SplitResult`, and we both have different metrics in the subclasses.